src/load_map.rs

```diff
@@ -48,4 +192,10 @@ mod tests {
         let serialized = serde_json::to_string(&st).unwrap();
         info!("{}", serialized);
     }
+  
+    pub fn test_1a_and_json_match() {
+        let code1a = tile_1a();
```

Comment:
Oh sorry I did a few things:

- Made tests run as part of the pipeline.
- Moved tests into their own module.

But I cooked it, `tile_1a()` here should be `super::tile_1a()` now. Idk if I can pull your PR and fix it.

Reply:
i'llll try to take a look